module/Entra/config/dependencyMapping.json

```diff
@@ -1,10 +1,10 @@
 {
-    "Microsoft.Entra.Users":["Microsoft.Graph.Users","Microsoft.Graph.Users.Actions","Microsoft.Graph.Users.Functions"],
+    "Microsoft.Entra.Users":["Microsoft.Graph.Users","Microsoft.Graph.Users.Actions","Microsoft.Graph.Users.Functions","Microsoft.Entra.Authentication"],
```

Comment:
Why do we keep repeating `Microsoft.Entra.Authentication`? We know that all sub-modules require `Microsoft.Entra.Authentication`. We should update our generatjon code to ensure `Microsoft.Entra.Authentication` is a dependent module in all sub-modules.

See how Graph SDK does
https://github.com/microsoftgraph/msgraph-sdk-powershell/blob/65b8e76e7119ad306e55cd9568669d66defc1c32/tools/GenerateMetaModule.ps1#L84-#L100